sshcode.go

```diff
@@ -608,3 +629,27 @@ func parseGCPSSHCmd(instance string) (ip, sshFlags string, err error) {
 
 	return strings.TrimSpace(userIP), sshFlags, nil
 }
+
+// When the user passes a path such as `/Workspace`, msys2 gives sshcode `C:/msys64/Worksapce`, which is not a valid remote path
+func gitbashWindowsDir(dir string) string {
+
+	// if dir is left empty, line82:main.go will set it to `~`, this makes it so that
+	// if dir is `~`, return `~/` instead of continuing with the gitbashWindowsDir()
+	// function. this prevens windows from trying to send a litteral instead of a relative
+	if dir == "~" {
+		return "~/"
+	}
```

Comment:
```suggestion
	// Don't bother figuring out path if it's relative to home dir.
	if strings.HasPrefix(dir, "~") {
		return dir
	}

```

Reply:
this edit won't work, and will cause issues because then someone can simply send `~` and windows/MinGW will interpret it as a literal, aka `'~'` and try and open `$PWD/'~'` instead of `$HOME`

this detects if the path is `~` and makes it `~/` so it sends a resolvable path

Reply:
```go
	// Don't bother figuring out path if it's relative to home dir.
	if strings.HasPrefix(dir, "~/") {
		if dir == "~" {
			return "~/"
		}
		return dir
	}
```
this work for you?